components/log-ingestor/src/aws_client_manager.rs

```diff
@@ -47,6 +51,18 @@ impl SqsClientWrapper {
     pub const fn from(client: SqsClient) -> Self {
         Self { client }
     }
+
+    pub async fn create(region: &str, access_key_id: &str, secret_access_key: &str) -> Self {
+        let sqs_endpoint = format!("https://sqs.{region}.amazonaws.com");
+        let sqs_client = create_sqs_client(
```

Comment:
```suggestion
        let sqs_client = sqs::create_new_client(
```

How about not renaming function at the top? Same for s3